File: 30DaysOfCode/day02_operators/operators.php
<?php
$handle = fopen ("php://stdin","r");
$mealCost = fgets($handle);
$tip = fgets($handle);
$tax = fgets($handle);

$total = round($mealCost + (0.01 * $tip * $mealCost) + (0.01 * $tax * $mealCost));

echo "The total meal cost is " . $total . " dollars.";
?>

File: 30DaysOfCode/day02_operators/operators.pl
my $mealCost = <STDIN>;
my $tip = <STDIN>;
my $tax = <STDIN>;

my $total = int($mealCost + (0.01 * $tip * $mealCost) + (0.01 * $tax * $mealCost) + 0.5);

print "The total meal cost is ", $total, " dollars.";
